users/vieting/experiments/librispeech/librispeech_960_pretraining/wav2vec2/config_02_fairseq_phoneme.py

```diff
@@ -94,6 +94,34 @@ def run_fairseq_pretraining_negatives_other_target():
     tk.register_output(f"{prefix_name}/{exp_name}/pretraining/scores.png", job.out_plot_se)
     return job
 
+def run_fairseq_pretraining_negatives_hard():
+    prefix_name = "experiments/librispeech/librispeech_960_pretraining/wav2vec2/"
+    alignment = get_alignment_hdf()
+    num_gpus = 8
+    itc_python_launcher = "/home/pv653172/setups/librispeech/20230328_wav2vec2/dependencies/python_launcher.sh"
```

Comment:
Is this used? We shouldn't need it anymore after switching to using apptainer.

Reply:
No, forgot to remove that